libsolidity/parsing/Parser.cpp

```diff
@@ -466,6 +466,7 @@ Parser::FunctionHeaderParserResult Parser::parseFunctionHeader(bool _isStateVari
 	RecursionGuard recursionGuard(*this);
 	FunctionHeaderParserResult result;
 
+	result.isVirtual = false;
```

Comment:
This line isn't really necessary anymore now that we have it member-initialized... Do you still want to keep it? For this line I'd be fine with keeping it...
Actually the line below is worse and what caught my eye here :-), because it suggests ``result.overrides`` is a plain pointer, which it isn't... it so happens that ``shared_ptr`` has a constructor for ``nullptr_t``, so it works, but if anything, I'd rather use ``result.overrides = {};`` or ``result.overrides.reset();``... or actually remove the line...

So yeah, I'd suggest to remove this line and the one below altogether... what do you think?

Reply:
I am fine with removing both lines.

Reply:
updated